salesforce_bulk/salesforce_bulk.py

```diff
@@ -172,18 +172,27 @@ def create_delete_job(self, object_name, **kwargs):
         return self.create_job(object_name, "delete", **kwargs)
 
     def create_job(self, object_name=None, operation=None, contentType='CSV',
-                   concurrency=None, external_id_name=None):
+                   concurrency=None, external_id_name=None, pk_chunking=False):
         assert(object_name is not None)
         assert(operation is not None)
 
+        extra_headers = {}
+        if pk_chunking:
+            if pk_chunking is True:
+                pk_chunking = u'true'
```

Comment:
I don't know if it matters, but the [doc](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/async_api_headers_enable_pk_chunking.htm) uses `TRUE` in capital letters.

Reply:
I don't think it does because the example page has all lower case: https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/asynch_api_code_curl_walkthrough_pk_chunking.htm